jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -502,11 +521,11 @@ async def put(self):
             )
 
 
-class HealthHandler(BaseHandler):
+class HealthHandler(APIHandler):
```

Comment:
i think we can probably just remove this. this was in place before to monitor the health of the mount-server

Reply:
Ah that makes sense why it wasn't hooked up as a handler anymore. I think its still useful to see the health of our backend server, unless we already have a way to do that? I know we technically have the GET /config but this feels a bit nicer/consistent.

Reply:
what is the benefit of having this over `GET /config` which also provides the cluster status? i think it might be a good move to move that over to be the new health check but it's not clear to me what we gain from having both.

Reply:
`GET /health` doesn't contain any code which could error, while `GET /config` does. One possible error state is if we introduce some bug within the pachyderm_sdk that raises a `ValueError`/`TypeError`/etc. Using the `GET /config` as a health indicator would indicate that the server is unhealthy/down when it's actually up.

I can see the argument of "is the server actually healthy if `GET /config` is broken" but I think it's reasonable to separate out these concerns.

Reply:
i suppose, but why not just catch those exceptions and respond as another state, one that indicates the backend is up but unhealthy?

Reply:
yeah, i think consolidating get config and health check that way seems like a good solution to me to have all health check/backend status available within the same endpoint. i don't feel that strongly about it so we can keep the health check separate too if you think that's a better idea.

Reply:
No I think that's reasonable. I'll add a new ticket for this. There's no harm in the way it's set up at the moment and I think adding another status will involve a couple of FE changes as well, so best to do it separately.

Reply:
Ticket: [INT-1198](https://pachyderm.atlassian.net/browse/INT-1198)